index.html

```diff
@@ -55,7 +55,12 @@
           <li class="nav-item">
             <a href="#portfolio" class="nav-link smoothScroll">Sponsors</a>
           </li>
-
+ 
+          
+          <li class="nav-item">
+            <a target="_blank" href="https://redmondstemcenter.org/summercamps.html" rel="noreferrer noopener" class="nav-link smoothScroll">Summer Camps</a>
```

Comment:
it's generally good to have this attribute, but it's not necessarily required in this context as we own the target link and so it's fine if e.g the target has the referrer header attached